RELEASE.md

```diff
@@ -14,6 +14,7 @@ Please follow the established format:
 
 ## Bug fixes and other changes
 - Fix improper display of 'run-command' inside the metadata panel. (#1569)
+- Replace semver with packaging. (#1578)
```

Comment:
I think we need to add the other section here: `## Bug fixes and other changes`.